### PR TITLE
Fix PIC_FLAG in run.d

### PR DESCRIPTION
**Pull request by marler8997**
This is the only place that uses `PIC_FLAGS`, everything else uses `PIC_FLAG`.  And the code is appending `PIC_FLAG` to `REQUIRED_ARGS` without a space.

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by marler8997** [comment]
We should have @wilzbach review before we merge.  He might catch something.

**Comment by wilzbach** [comment]
Holy moly. Can we please stop merging things _a minute_ after they were opened?

**Comment by marler8997** [comment]
Lol...one extreme to the other :)

File: test/run.d
Comment (wilzbach) [inline test/run.d]: This seems needlessly complicated. Why not `text (left, " ", right)` ?

Comment (marler8997) [reply test/run.d]: That works so long as you don't care about the extra space when one argument is empty.

**Comment by marler8997** [comment]
Should be good to go on this one.

**Comment by Geod24** [comment]
> Holy moly. Can we please stop merging things a minute after they were opened?

Please, yes. Can we build that rule into dlang-bot ?

**Comment by marler8997** [comment]
> Please, yes. Can we build that rule into dlang-bot ?

In this case, I don't think developing a tool to police the people with write permissions to this repo makes alot of sense.  For most things a simple conversation will suffice.

There would be a lot of potential for problems when implement this particular rule. Which time do you take?  The time the pull request was opened?  The time of the commit?  Maybe someone closed and reopened the pull request, do you take the new open time or the original open time?  What times does github provide?  How do you reconcile the timestamps between github and dlang-bot?  How much time should you require?  If auto-merge was added too soon, do you just wait a while and then merge it, or report a failure and remove the label? This also adds another point of failure in dlang-bot when it fails to request the times of the PR.  Note that github limits the amount of requests you can do for it's data. There may be times when it makes sense to add auto-merge immediately, so do you make a way to disable the rule?

The way I see it, you could try to design a robust solution for this and add the burden of maintaining it forever, or you could just try to communicate with your peers.

Of course, I think there are cases where these types of rules make sense.  Linters are good since it's very tedious and error prone to expect reviewers to check code style.

**Comment by Geod24** [comment]
@marler8997 : It's not about policing. PR merged instantly are usually trivial, or seen as such. The way I see it, a simple "PR had to be open for more than 24 hours for merge to happen" would be a good first step. So you can still set `auto-merge` on trivial PRs, but people have time to object before the actual merge.
Also, having `72h no objection => merge` being automated would fall in the same mindset.

**Comment by marler8997** [comment]
I think in general it's better to avoid "red tape" and "complexity".  What you're talking about adds more complexity to the merge system.  Does it add a benefit? Sure I can agree with that.  But it also comes with a cost and you have to weigh that against any alternative solution.  The alternative solution in this case is to talk to the maintainer.  Let them know they shouldn't merge PRs right away so others get a chance to review.  Then we don't need to add more work for the community to maintain a system to enforce this rule.

**Comment by wilzbach** [comment]
I agree with Jonathan here. There are only a handful people entrusted with merge rights. If we lay out clear common guidelines and put them in a e.g. MAINTAINERS.md (for future references), I think it's more than reasonable to expect that the maintainers will follow them.

**Pull request merged**
